libs/ui/src/virtual_keyboard/scan_panel_virtual_keyboard.test.tsx

```diff
@@ -0,0 +1,245 @@
+import userEvent from '@testing-library/user-event';
```

Comment:
Most tests in this file are adapted from `virtual_keyboard.test.tsx` unless noted